internal/app/secretless/handlers/mysql/handler.go

```diff
@@ -89,37 +128,69 @@ func stream(source, dest net.Conn, callback func([]byte)) {
 	}
 }
 
-// Pipe performs continuous bidirectional transfer of data between the client and backend.
-func (h *Handler) Pipe() {
+// pipe performs continuous bidirectional transfer of data between the client and backend.
+//
+func (h *Handler) pipe() {
 	plugin_v1.PipeHandlerWithStream(h, stream, h.EventNotifier, func() {
 		h.ShutdownNotifier(h)
 	})
 }
 
-// Run configures the backend connection info, connects to the backend to
-// complete the connection phase, and pipes the data between the client and
-// the backend
+// This is the main handler method. It:
```

Comment:
I would guess that `go lint` would complain about this comment not starting with the (exportable) method name: `Run`